### DISCUSSION TITLE
Kugou Music icon

### DISCUSSION THREAD

**Discussion by maybeSilence** [elrumo/macOS_Big_Sur_icons_replacements#1460] [Icon request]
Can you design the Kugou music icon? Kugou Music has a lot of users in China, and I have been looking for a long time and have not found a Mac icon about this app。
![kugou-music](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/52225423/7135d2f6-8bfc-4187-b540-17290f44e959)

**Comment by Karukhai** [comment]
Hi, I formatted the logo in this style. You can find it by searching or inside my profile: https://macosicons.com/#/u/kenji
